Resources/public/js/views/services/plugins/ez-locationsearchplugin.js

```diff
@@ -5,280 +5,18 @@
-                            return;
-                        }
-
-                        locationStructArr[i].contentType = contentType;
-                    });
-
-                contentType.load(options, endContentTypeLoad);
-            }, this);
-
-            tasks.done(function () {
-                callback(loadContentTypeError, locationStructArr);
-            });
-        },
-
-        /**
-         * Loads Content for each location struct in given array and puts it in the new `content`
-         * field for the location struct.
-         *
-         * @method _loadContentForLocations
-         * @protected
-         * @param {Array|Null} locationStructArr
-         * @param {Function} callback
-         */
-        _loadContentForLocations: function (viewName, locationStructArr, callback) {
-            var contentService = this.get('host').get('capi').getContentService(),
-                contentIds,
-                Content = this.get('contentModelConstructor'),
-                query;
-
-            contentIds = Y.Array.reduce(locationStructArr, '', function (previousId, struct) {
-                return previousId + "," + struct.location.get('contentInfo').get('contentId');
-            });
-
-            query = contentService.newViewCreateStruct('contents-loading-' + viewName, 'ContentQuery');
-            query.body.ViewInput.ContentQuery.Criteria = {
-                "ContentIdCriterion": contentIds
-            };
-
-            contentService.createView(query, function (err, response) {
-                if (err) {
-                    callback(err, locationStructArr);
-                    return;
-                }
-                Y.Array.each(response.document.View.Result.searchHits.searchHit, function (hit, i) {
-                    locationStructArr[i].content = new Content();
-                    locationStructArr[i].content.loadFromHash(hit.value.Content);
-                });
-                callback(err, locationStructArr);
-            });
-        },
-    }, {
-        NS: 'locationSearch',
-
-        ATTRS: {
-            /**
-             * Holds the eZ.Location constructor function
-             *
-             * @attribute locationModelConstructor
-             * @type {Function}
-             * @default Y.eZ.Location
-             */
-            locationModelConstructor: {
-                value: Y.eZ.Location
-            },
-
-            /**
-             * Holds the eZ.Content constructor function
-             *
-             * @attribute contentModelConstructor
-             * @type {Function}
-             * @default Y.eZ.Content
-             */
-            contentModelConstructor: {
-                value: Y.eZ.Content
-            }
-        },
-    });
+    console.warn('[DEPRECATED] ez-locationsearchplugin module and Y.eZ.Plugin.LocationSearch are deprecated');
+    console.warn('[DEPRECATED] ez-locationsearchplugin module and Y.eZ.Plugin.LocationSearch will be removed in PlatformUI 2.0');
```

Comment:
it's already the case in line 18 + in the doc of the module in the beginning of the file. But I agree with @yannickroger , when we'll effectively remove the module, we'll have to document that.